packages/core/medusa-test-utils/src/events.ts

```diff
@@ -0,0 +1,28 @@
+import { IEventBusModuleService } from "@medusajs/types"
+
+// Allows you to wait for all subscribers to execute for a given event. Only works with the local event bus.
+export const waitSubscribersExecution = (
+  eventName: string,
+  eventBus: IEventBusModuleService
+) => {
+  const subscriberPromises: Promise<any>[] = []
+
+  ;(eventBus as any).eventEmitter_.listeners(eventName).forEach((listener) => {
+    ;(eventBus as any).eventEmitter_.removeListener("order.created", listener)
```

Comment:
```suggestion
    ;(eventBus as any).eventEmitter_.removeListener(eventName, listener)
```

Reply:
Oh how did I miss that, I thought we changed it in your commit??

Reply:
I thought too 😂

Reply:
have you forced push at some point?

Reply:
yeah after the rebase, maybe that's why...

Reply:
If I can recommend that you always use force with lease instead 😄